scripts/realm-init.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+set -o errexit
+
+function checkIfKeycloakRunning {
+    echo "Checking if keycloak is running..."
+    STAT=`netstat -na | grep 8080 | awk '{print $6}'`
+    if [ "$STAT" = "LISTEN" ]; then
+        echo "Keycloak port accessible and detected"
+        sleep 10
+        installRealm
+    elif [ "$STAT" = "" ]; then 
+        sleep 5
+        echo "Keycloak is not running:`$STAT`. Trying again."
+        checkIfKeycloakRunning
+    fi
+}
+ 
+function installRealm {
```

Comment:
`importRealm` might be a better description to keep it consistent with keycloak docs.